content/guides/output-management.md

```diff
@@ -3,85 +3,262 @@ title: Output Management
-    path: path.resolve(__dirname, 'dist'),
-    filename: '[name].bundle.js'
-  },
-
-  plugins: [
-    new HtmlWebpackPlugin({
-      title: 'Output Management',
-      favicon: './favicon.ico'
-    })
-  ]
-};
+``` bash
+Hash: 3adf5eed79147a35559e
+Version: webpack 2.6.1
+Time: 844ms
+           Asset       Size  Chunks                    Chunk Names
+vendor.bundle.js     544 kB       0  [emitted]  [big]  vendor
+   app.bundle.js    2.81 kB       1  [emitted]         app
+      index.html  244 bytes          [emitted]
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 172 bytes {1} [built]
+   [4] multi lodash 28 bytes {0} [built]
+Child html-webpack-plugin for "index.html":
+       [0] ./~/lodash/lodash.js 540 kB {0} [built]
+       [1] ./~/html-webpack-plugin/lib/loader.js!./src/index.ejs 540 bytes {0} [built]
+       [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+       [3] (webpack)/buildin/module.js 517 bytes {0} [built]
 ```
 
-This setup will generate the same HTML shown in the example above, excluding the extracted CSS. The plugin allows for many settings including extending your own template, minifying the output, and changing the script injection site. See [its documentation](/plugins/html-webpack-plugin) for more details.
+Open `index.html` in your code editor again. This time you will see that the `HtmlWebpackPlugin` has dynamically added a title (which we specified in our `webpack.config.js` at the `HtmlWebpackPlugin` configuration options), as well as the generated bundles.
 
-T> Check out the [`HTMLWebpackTemplate`](https://github.com/jaketrent/html-webpack-template) extension for even more options including easy insertion of an `appMountId`, `meta` tags, and a `baseHref`.
+This is great, because now we don't have to manually adjust our `index.html` file anymore, webpack takes care of that. As an added bonus, we now keep the `index.ejs` template in our `src` directory, so our `dist` directory only contains generated files.
 
+T> Check out the [`HtmlWebpackTemplate`](https://github.com/ampedandwired/html-webpack-plugin#configuration) page for more advanced options, including outputting multiple HTML files.
 
-## Multiple HTML Files
 
-To generate multiple HTML files, say for a multi-page web application, you can utilize the [`MultipageWebpackPlugin`](https://github.com/mutualofomaha/multipage-webpack-plugin). This plugin is similar to the `html-webpack-plugin`, in fact it uses that plugin under the hood, however it can be used to generate an HTML file per entry point.
+## Cleaning up the `/dist`folder ##
 
+As you might have noticed over the past guides and code example, our `/dist` folder has become quite cluttered. Webpack will generate the files and put them in the `/dist` folder for you, but it doesn't keep track of which files are actually in use by your project.
 
-## The Manifest
```

Comment:
I honestly thought it was way too advanced to be useful, but I'll add it back in with some adjustments and link off to the right resources in case people are interested in it.